uitools/import/Esri/ArcGISRuntime/Toolkit/FloorFilter.qml

```diff
@@ -89,19 +90,40 @@ Control {
                 orientation: Qt.Horizontal
             }
 
-            Repeater {
-                id: repeater
-                property int downItem
-
-                model: controller.levels
-                delegate: ToolButton {
-                    visible: !closer.checked
-                    checked: controller.selectedLevelId === model.modelId
-                    autoExclusive: true
-                    Layout.fillWidth: true
-                    text: collapser.checked ? model.shortName : model.longName
-                    onClicked: {
-                        controller.selectedLevelId = model.modelId
+            Flickable {
+                visible: !closer.checked
+
+                // dont need to use the id of the column
+                contentHeight: contentItem.childrenRect.height
+                Layout.fillWidth: true
+                height: repeater.buttonHeight * maxNumberLevels
+                clip: true
+                ScrollBar.vertical: ScrollBar {}
+
+                Column {
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    Component.onCompleted: console.log("width col", width)
+                    Repeater {
+                        id: repeater
+
+                        property int downItem
+                        property int buttonHeight: 50
```

Comment:
using a property to decide the buttomHeight. Could let it handled by the delegates, setting this property by delegates `onCompleted` event (lots of useless calls since I believe they will have all same height).
Possible problem: if the buttons change their height (may be a screen resize will trigger it?), their `height` and `buttomHeight` will not match anymore